test/machineslib.py

```diff
@@ -198,6 +198,9 @@ def setUp(self):
             # https://bugs.launchpad.net/ubuntu/+source/libvirt-dbus/+bug/1892757
             m.execute("usermod -a -G libvirt libvirtdbus")
 
+        # we don't have configuration to open the firewall for local libvirt machines, so just stop firewalld
+        m.execute("systemctl stop firewalld")
+
         self.startLibvirt()
```

Comment:
So that means even *starting* libvirt fails with firewalld running? This is a release-critical bug,  can you please report it?

(Also: We should really figure this out properly, and fix c-machines with firewalld running)

Reply:
ok removed the commit that does that - and will make a naughty for it once I figure this out what exactly made the test fail

Reply:
Sorry for the misunderstanding -- my "requested changes" was just for the rebase. But sure, we can land this PR without a green f35, and fix that separately.